ArduCopter/motor_test.cpp

```diff
@@ -51,6 +51,11 @@ void Copter::motor_test_output()
         // calculate pwm based on throttle type
         switch (motor_test_throttle_type) {
 
+            case MOTOR_TEST_COMPASS_CAL:
+                compass.set_voltage(battery.voltage());
```

Comment:
Any reason to do this here? Without reading the battery again, this should always be setting the same value.